.circleci/config.yml

```diff
@@ -208,9 +208,44 @@ commands:
 
   run_build:
     steps:
+      # TODO: remove the line below
+      # Before merging this PR we need to use an alternative develop branch to test the functionality, `develop-cache-test`
```

Comment:
We want to be able to run without ccache on tags, so we still need to keep a non-cached variant of this command around.